preview-src/index.ts

```diff
@@ -131,14 +133,19 @@ function appendComment(comment: any) {
 
 function updateCheckoutButton(isCheckedOut: boolean) {
 	const checkoutButton = (<HTMLButtonElement>document.getElementById(ElementIds.Checkout));
+	const checkoutMasterButton = (<HTMLButtonElement>document.getElementById(ElementIds.CheckoutMaster));
 	checkoutButton.disabled = isCheckedOut;
-	checkoutButton.innerHTML = isCheckedOut ? `Checked Out` : `Checkout`;
+	checkoutMasterButton.disabled = false;
+	const activeIcon = '<svg class="octicon octicon-check" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>';
```

Comment:
I think we need more handling for light themes, the icon and text color match the background on Light+

Reply:
We can probably add 

```css
.vs .octicon {
	color: white;
}
.vs-dark .octicon {
	color: black;
}
```

to the index.css

Reply:
Yikes, I missed that. I'll use `var(--vscode-foreground)` for the text color instead. Thanks for catching that!